README.md

```diff
@@ -122,4 +122,15 @@ In most situations it is possible that not all Freesound services need to be run
 
 ```
 docker-compose up web search
-```
\ No newline at end of file
+```
+
+
+### Running tests
+
+You can run tests using the Django test runner in the `web` container like that:
+
+```
+docker-compose run --rm web python manage.py test --setings=freesound.test_settings.py  accounts apiv2 bookmarks donations follow forum general geotags messages monitor ratings search sounds support tags tickets utils wiki
```

Comment:
do we need to specify all of the modules? Django should run all of them if we specify nothing?

Reply:
I thought it would do 3rd party tests as well, but I just tried and it seemed to work fine so I'll remove the comment and app names